src/main/service/filterService.ts

```diff
@@ -1,11 +1,13 @@
 import {Court} from '../models/court';
 
+const filterNames = ['Jurisdiction', 'Region'];
+
 export class FilterService {
-  public getFilterValueOptions(filterName: string, list: Array<Court>): string[] {
+  private getFilterValueOptions(filterName: string, list: Array<Court>): string[] {
```

Comment:
by making this private it is not accessible in tests
My linter is screaming  Property 'getFilterValueOptions' is private and only accessible within class 'FilterService' when ac

Reply:
i set it to private as it it not needed outside this class, it is only used within filter service, its unit tests are covered by publicly accessible tests